tests/integration/models/reddit/test_subreddit.py

```diff
@@ -1473,6 +1474,99 @@ def test_submissions__with_pause_and_skip_after(self, _):
             assert submission_count < 100
 
 
+class TestSubredditModerationStreams(IntegrationTest):
+    def setup_reddit(self):
+        self.reddit = Reddit(
+            client_id=pytest.placeholders.client_id,
+            client_secret=pytest.placeholders.client_secret,
+            refresh_token=pytest.placeholders.refresh_token,
+            redirect_uri=pytest.placeholders.redirect_uri,
+            user_agent=pytest.placeholders.user_agent,
+        )
+
+    @property
+    def subreddit(self):
+        return self.reddit.subreddit(pytest.placeholders.test_subreddit)
+
+    @mock.patch("time.sleep", return_value=None)
+    def test_edited(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+            "TestSubredditModerationStreams.edited"
```

Comment:
Each time you use a cassette, it should be named `"ClassName.method_name"`. For this one, please change it to be `"TestSubredditModerationStreams.test_edited"`. The cassettes will also need to be renamed, but you don't have to re-record them (unless for some reason you want to).